recipe/meta.yaml

```diff
@@ -8,19 +8,19 @@ source:
   fn: pytest-django-{{ version }}.tar.gz
   url:  https://pypi.io/packages/source/p/pytest-django/pytest-django-{{ version }}.tar.gz
   sha256: 038ccc5a9daa1b1b0eb739ab7dce54e495811eca5ea3af4815a2a3ac45152309
+  patches:
+    # setup_requires is throwing errors in setuptools
+    - setuptools_scm.patch
 
 build:
-  number: 0
-  preserve_egg_dir: True
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  entry_points:
-    - pytest11 = django:pytest_django.plugin
```

Comment:
Was this entry point mean to be dropped?

Reply:
Yes it was. It still works as intended. As I understand it `pytest` can discover their plugins using the `entry_points` system but they are not actual runnable functions and shouldn't have binaries created for them.